test/basic.js

```diff
@@ -52,4 +53,143 @@ describe('Basic Tests', () => {
       .get('/404') // As long as the page 404 doesn't exist, this will give a 404
       .expect(404, done);
   });
+
+  // Putting semester reset tests in this file because they don't make sense anywhere else
+  describe('Semester reset tests', () => {
+    beforeEach((done) => {
```

Comment:
Move this up a level in the test file - it should be under the first describe instead of this one